pkg/kubelet/kubelet_pods.go

```diff
@@ -185,7 +185,10 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 				return nil, cleanupAction, fmt.Errorf("volume subpaths are disabled")
 			}
 
-			if filepath.IsAbs(subPath) {
+			// On Windows, filepath.IsAbs does not consider \ or / prefixed paths as absolute, even though
+			// they are treated as such.
+			isWindowsAbs := runtime.GOOS == "windows" && (strings.HasPrefix(subPath, `/`) || strings.HasPrefix(subPath, `\`))
```

Comment:
should this logic has a separate method for validation? In a PR description I see something about other place having (or needing to have) the same logic. Would be great to cross-reference those or use the same method in both cases.

> even though in the end they are treated as such. Because of this, absolute paths can be mounted as volume subpaths in Windows.